generated/explanations/euler.md

```diff
@@ -0,0 +1,62 @@
+Burning is to delete one set of “borrow” and “deposit”, that is to repay one borrow position and withdraw the respective collateral i.e deleting one set of deposit and borrow the multiple sets of deposits and borrows made. 
+
+#### Sub Accounts
+Euler also has a very protocol specific feature called sub-accounts. Once an account is borrowed in an isolated tier, it would be locked from borrowing other assets. In order to not limit the user, Euler enables every Ethereum account to create up-to 256 sub-accounts, allowing the user to segregate their liabilities according to their needs.
+
+#### Transaction Builder
+Euler allows users to bundle a bunch of actions that would otherwise be separate transactions on their own into one by using the transaction builder. This feature ties in with the defer liquidity feature where the liquidity check on the user is done after a bundle of transactions considered as one, whereas usually, a liquidation  check is performed after every action that requires user liquidity (Borrow, withdraw etc). 
+
+#### Feeless Flash Loans
+Euler lacks a native concept of flash loans. But by making use of the defer liquidity feature, users can make an uncollateralized borrow, perform whatever operation they like, and then repay the borrow - all without being charged extra fees. This can be used to rebalance positions, build-up leveraged positions, take advantage of external arbitrage opportunities, and more. Because Euler only charges fees according to the time value of money, and from the blockchain's perspective flash loans are held for a duration of 0 seconds, they are entirely free on Euler (ignoring gas costs).
+
+#### Interest Rates
+To improve capital efficiency and do away with errors in determining parameters for the interest rates of every lending market, Euler uses reactive interest rates. By using a PID controller, Euler amplifies or dampens the rate of change in interest rates when utilization is above or below the target level of utilization respectively. As a result, reactive interest rates  that adapt to market conditions for the underlying asset in real-time are created, without the need for ongoing governance intervention. Interest is accrued every second on Euler.
+
+ 
+ **EUL token**        
+
+## EUL
+The total supply of EUL is 27,182,818 (in homage to Euler’s number,[ e](https://en.wikipedia.org/wiki/E_(mathematical_constant))). The total supply of EUL is fixed for the first 4 years, after which EUL token holders may enact a governance proposal to inflate the supply by a maximum 2.718% per year. In that scenario, newly minted EUL will enter circulation via the Treasury.
```

Comment:
Please check the formatting for the EUL token step

https://dodao.academy/course/file/view/lending-and-borrowing-in-defi/euler/explanation/eul-Token

also the keys should be all small case `eul-Token` -> `eul-token`